app/controllers/projects_controller.rb

```diff
@@ -70,12 +70,21 @@ def update
     update! do |format|
       format.html do
         if resource.errors.present?
-          flash[:alert] = resource.errors.full_messages.to_sentence
+          error_message = ''
+          resource.errors.each do |error|
+            error_message += resource.errors[error][0]
+            error_message += '<br/>'
+          end
```

Comment:
We should move this method to a separate class that we can use to decorate errors.